src/main/java/com/utsusynth/utsu/engine/Engine.java

```diff
@@ -189,9 +189,19 @@ public PlaybackStatus getStatus() {
         if (!notes.hasNext()) {
             return Optional.absent();
         }
+        
         int totalDelta = notes.getCurDelta(); // Absolute position of current note.
         Voicebank voicebank = song.getVoicebank();
-        boolean isFirstNote = true;
+        boolean isFirstNote = true;        
+        final File finalSong; // Is the final keyword causing file locking??
+
+        try {
+            finalSong = File.createTempFile("utsu-", ".wav");
+            finalSong.deleteOnExit(); // Required??
```

Comment:
I don't think this is required since the temp directory is already deleted on shutdown.